### DISCUSSION TITLE
[cosmetic] Desktop OS dark mode leaking to auto-complete in Superset text boxes

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#19197] [Ideas]
<img width="1671" alt="Screen Shot 2021-07-12 at 10 24 44 PM" src="https://user-images.githubusercontent.com/67837651/125496103-9ac6473d-4267-4644-a563-93d792b3c189.png">

<img width="1484" alt="Screen Shot 2021-07-12 at 3 37 09 PM" src="https://user-images.githubusercontent.com/67837651/125492462-71a33aab-788e-4f92-a3dc-78d776147e8b.png">

Search dropdown background color black - (which used to be white or none) is added and width is cut.

**Comment by rusackas** [comment]
This seems to be your computer's dark mode setting correctly applying itself to chrome, which in turn applies this styling to native input elements.

**Comment by junlincc** [comment]
is it the only thing that turns black in the product when dark mode is on? lol.. if that's the case, can we have our product dark-mode proof for the sake of consistency?

**Comment by rusackas** [comment]
It's possible, at least for some browsers. Styling these things gets a little weird. I would say it's a high-risk, low ROI change, until we have visual regression testing in place to make sure we don't break something someplace else in the UI by accident.

**Comment by junlincc** [comment]
p9 makes sense lol

**Comment by srinify** [comment]
Converting to Discussion since it's a feature request!
